docs/pages/database-access/guides/rds.mdx

```diff
@@ -43,74 +53,294 @@ This guide will help you to:
 
+### Prepare your environment
+
+Next, get your environment ready to run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Host">
+
 Install Teleport on the host where you will run the Teleport Database Service:
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Create the Database Service configuration:
+</TabItem>
+<TabItem label="Kubernetes Cluster">
 
-<ScopedBlock scope={["oss", "enterprise"]}>
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
 
-```code
-$ teleport db configure create \
-   -o file \
-   --proxy=teleport.example.com:3080 \
-   --token=/tmp/token \
-   --rds-discovery=us-west-1
-```
+</TabItem>
+</Tabs>
 
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
+### Create a configuration file
+
+Generate a configuration for the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Host">
+
+Run the command below on your Linux host.
+
+Assign <Var name="teleport.example.com:443" /> to the host and port of your
+Teleport Proxy Service or Enterprise Cloud/Team tenant. Assign <Var
+name="us-west-1" /> to the region associated with the RDS databases you would
+like Teleport to discover.
 
 ```code
 $ teleport db configure create \
    -o file \
-   --proxy=mytenant.teleport.sh:443 \
+   --proxy=<Var name="teleport.example.com:443" />  \
    --token=/tmp/token \
-   --rds-discovery=us-west-1
+   --rds-discovery=<Var name="us-west-1" />
 ```
 
-</ScopedBlock>
-
 The command will generate a Database Service configuration with RDS/Aurora
-database auto-discovery enabled on the `us-west-1` region and place it at the
+database auto-discovery enabled on the configured region and place it at the
 `/etc/teleport.yaml` location.
 
-## Step 3/6. Create an IAM policy for Teleport
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+
+Follow the instructions below on your local workstation.
+
+Create a file called `kube-agent-config.yaml` in your current working directory
+with the following content:
+
+```yaml
+version: v3
+teleport:
+  nodename: teleport-rds
+  data_dir: "/var/lib/teleport"
+  proxy_server: ""
+  join_params:
+    method: token
+    token_name: /etc/teleport-secrets/auth-token
+db_service:
+  enabled: true
+  aws:
+  - types: ["rds"]
+    regions:
+    - ""
+    tags:
+      "*": "*"
+auth_service:
+  enabled: "no"
+ssh_service:
+  enabled: "no"
+proxy_service:
+  enabled: "no"
+```
+
+Assign `teleport.proxy_server` to the host and port of your Teleport Proxy
+Service or Enterprise Cloud/Team tenant (e.g., `mytenant.teleport.sh:443`).
+Assign `db_service.aws[0].regions` to a list of AWS region strings (e.g.,
+`"us-west-1"`) in which you would like the Teleport Database Service to discover
+databases.
+
+The Helm chart that we will  use to install the Teleport Database Service mounts
+a secret containing the join token we generated earlier to
+`/etc/teleport-secrets/auth-token`. The configuration above instructs the
+Database Service to search for its token there.
```

Comment:
Since we never explicitly create this secret or define a configuration that does, I assume that the helm chart is preset to look for `/tmp/token` and creates the secret if found. But you know what they say about assumptions, so I'd mentioning it.